.vscode/settings.json

```diff
@@ -64,5 +64,11 @@
       "changeProcessCWD": true,
       "directory": "./src",
     },
-  ]
+  ],
+  "go.useLanguageServer": true,
+  "go.languageServerExperimentalFeatures": {
+    "format": true,
+    "diagnostics": true,
+    "documentLink": true
+  }
```

Comment:
Just wondering....shouldn't this file be in our gitignore?

Reply:
Don't we use this to enable common eslint features?

Reply:
Hmmm...well yes we could. I havent pushed my recent changes to this file though. But I could do it as well. So then we'll keep this in the repo until we merge track2 into master? And then delete it before then? Or would be keep it?

Reply:
Well, if this is how we standardize on what eslint features to require then I think we'll have to keep it.

Reply:
Yes that's true. Ok so we're keeping it 👍